internal/service/alarms/internal/dictionary/dictionary.go

```diff
@@ -4,354 +4,81 @@ import (
 	"context"
 	"fmt"
 	"log/slog"
-	"sync"
+	"time"
 
+	"github.com/go-logr/logr"
 	"github.com/google/uuid"
 	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
-	"k8s.io/apimachinery/pkg/labels"
-	"k8s.io/apimachinery/pkg/selection"
-	clusterv1 "open-cluster-management.io/api/cluster/v1"
 	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
 
-	"github.com/openshift-kni/oran-o2ims/internal/controllers/utils"
-	"github.com/openshift-kni/oran-o2ims/internal/service/alarms/internal/clusterserver"
-	"github.com/openshift-kni/oran-o2ims/internal/service/alarms/internal/db/models"
 	"github.com/openshift-kni/oran-o2ims/internal/service/alarms/internal/db/repo"
-	"github.com/openshift-kni/oran-o2ims/internal/service/common/clients"
+	"github.com/openshift-kni/oran-o2ims/internal/service/alarms/internal/infrastructure"
+	"github.com/openshift-kni/oran-o2ims/internal/service/alarms/internal/infrastructure/clusterserver"
 	"github.com/openshift-kni/oran-o2ims/internal/service/common/clients/k8s"
 )
 
-const (
-	managedClusterVersionLabel = "openshiftVersion"
-	localClusterLabel          = "local-cluster"
-)
+const pollInterval = 30 * time.Minute
 
+// AlarmDictionary represents the dictionary of alarms
 type AlarmDictionary struct {
```

Comment:
I didn't notice this when it was first introduced, and I realize this isn't being changed in this PR, but is this a good name for this?   Seems confusing since this isn't actually an "alarm dictionary" ... maybe `AlarmDictionaryService` or something similar?